README.md

```diff
@@ -66,9 +68,27 @@ To reduce costs, you can switch to free SKUs for Azure App Service, Azure Cognit
 ⚠️ To avoid unnecessary costs, remember to take down your app if it's no longer in use,
 either by deleting the resource group in the Portal or running `azd down`.
 
-### Prerequisites
+### Project setup
+
+You have a few options for setting up this project. 
+The easiest way to get started is GitHub Codespaces, since it will setup all the tools for you,
+but you can also [set it up locally](#local-environment) if desired.
+
+#### GitHub Codespaces
 
-#### To run locally
+You can run this repo virtually by using GitHub Codespaces, which will open a web-based VS Code in your browser:
+
+[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=599293758&machine=standardLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestUs2)
+
+#### VS Code Remote Containers
+
+A related option is VS Code Remote Containers, which will open the container in your local VS Code:
```

Comment:
"open the container" - which container?
I'm personally not familiar with this

Reply:
Reworded to "A related option is VS Code Remote Containers, which will open the project in your local VS Code using the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers):"